Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -274,6 +275,13 @@ @interface WKContentView (ImageAnalysisPreview) <QLPreviewControllerDelegate, QL
 
 #endif // USE(QUICK_LOOK)
 
+#if HAVE(UI_TEXT_SELECTION_DISPLAY_INTERACTION)
```

Comment:
I'm assuming this was added elsewhere? Just asking because it's a different name than what we're using elsewhere in this file.